orchestrator/api/api_v1/endpoints/processes.py

```diff
@@ -386,11 +386,11 @@ async def websocket_process_detail(websocket: WebSocket, pid: UUID, token: str =
     await websocket_manager.connect(websocket, channel)
 
 
-def get_current_process_data(pid: UUID) -> dict[str, Any]:
+def get_current_process_data(pid: UUID) -> Dict[str, Any]:
```

Comment:
Why doe we revert to Dict and List (uppercased)?

Reply:
If we want to remain compatible with Python 3.8 we have to leave it like this, I enabled tests and linting for 3.8 and this tripped up. Furthermore, the pydantic models are still not completely compatible with the generics. Leaving this for now.